src/coreclr/nativeaot/Runtime/RhConfig.h

```diff
@@ -11,6 +11,8 @@
 // Values can also be embedded in the compiled binary.
 //
 
+#ifndef __RHCONFIG_H__
+#define __RHCONFIG_H__
```

Comment:
nit: by convention, __ prefixed / suffixed are system-defined macros:
```suggestion
#define RHCONFIG_H
```

Reply:
😮 I've been doing that since I learned C - presumably from an unfortunate example - and no one has ever pointed it out to me. Thank you.

Reply:
It learned it from @janvorli https://github.com/dotnet/runtime/pull/44505#discussion_r535709229. :)

Reply:
`#pragma once` works too. It is not part of the standard, but it is supported by every C/C++ compiler out there that matters.